JWBestPracticeApps/JWBestPracticeApps.xcodeproj/xcshareddata/xcschemes/JWCasting.xcscheme

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
why is this appearing in the diff ?

Reply:
That should be from reverting the commit where I removed it.

Reply:
if we're not changing anything in `JWCasting`, we shouldn't have changes in the scheme. Please Close this PR and start a new one with only the changes we need

Reply:
So the casting the demo and best practices apps both do not work with the Chromecast receivers in the office (the Android devices work fine). In exploring possible causes, I tried making the JWCasting scheme (tied to your user's xcuserdata) a shared scheme -- that is, I checked off "Shared" in the Manage Schemes menu. This made a **copy** of the scheme, which was not deleted even when I unchecked the "shared" option. That's why it's appearing in the diff.